collision.go

```diff
@@ -102,9 +139,13 @@ func (cs *CollisionSystem) Update(dt float32) {
 					e1.SpaceComponent.Position.Y += mtd.Y
 				}
 
+				collided = true
 				Mailbox.Dispatch(CollisionMessage{Entity: e1, To: e2})
 			}
 		}
+
+		// if no one collided reset to false
```

Comment:
I decided to remove the comment altogether. Doesn't clarify that much, and it's only used internally.